# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

ChooseGoalActionGoal action_goal
ChooseGoalActionResult action_result
ChooseGoalActionFeedback action_feedback
